react/features/base/conference/middleware.any.js

```diff
@@ -240,8 +241,15 @@ function _conferenceJoined({ dispatch, getState }, next, action) {
         dispatch(openDisplayNamePrompt(undefined));
     }
 
+    if (firstJoin) {
```

Comment:
I think this logic should go in features/conference. features/base is for foundational stuff, Salesforce is not that.